cognite/client/_api/extractionpipelines.py

```diff
@@ -0,0 +1,167 @@
+from typing import *
```

Comment:
is it `extractpipelines` or `extractpipeline` api? would be better to use the same verbiage throughout (`extractionpipelinerun` vs `extractionpipelinesrun`).

Reply:
ah its `extractpipelines`

Reply:
Our reasoning was: It's extraction pipelines (plural), just like assets, events, etc. It's extractionpipeline (singular) runs (plural) since it's runs for a single extraction pipeline.